contact.css

```diff
@@ -0,0 +1,320 @@
+  font-weight: bold;
+}
+
+@keyframes text_reveal_box {
+  50% {
+    width: 100%;
+    inset-inline-start: 0;
+  }
+  100% {
+    width: 0;
+    inset-inline-start: 100%;
+  }
+}
+
+@keyframes text_reveal {
+  100% {
+    color: white;
+  }
+}
+
+.page-title {
+  display: block;
+  font-size: 2rem;
+  width: fit-content;
+  margin-inline: auto;
+  margin-block-start: 5rem;
+  position: relative;
+  color: transparent;
+  animation: text_reveal 1s ease forwards;
+  animation-delay: 1.5s;
+}
+
+.page-title span {
+  position: absolute;
+  inset-block-start: 0;
+  inset-inline-start: 0;
+  height: 100%;
+  width: 0;
+  background-color: rgb(147, 77, 16);
+  animation: text_reveal_box 1s ease;
+  animation-delay: 1s;
+}
+
+.page-subtitle {
+  visibility: hidden;
+  padding-block-end: 3rem;
+}
+
+.section-form {
+  display: flex;
+  flex-direction: column-reverse;
+  justify-content: center;
+  align-items: center;
+}
+.text-contact {
+  color: white;
+  text-align: center;
+}
+
+#form-ul {
+  list-style: none;
+  margin: 0;
+  margin-block-end: 1rem;
+  padding: 0;
+}
+
+#form-ul li {
+  margin-block-end: 1rem;
+}
+
+form {
+  padding: 1rem;
+  border-radius: 0.4rem;
+}
+
+input,
+textarea,
+select {
+  font: inherit;
+  display: block;
+  width: 100%;
+  padding: 0.5rem;
+  border-radius: 0.25rem;
+  box-sizing: border-box;
+  background-color: rgb(225, 222, 222);
+  border: transparent;
+  margin-inline: auto;
+}
+
+textarea {
+  resize: none;
+  width: 100%;
+}
+
+input:focus,
+textarea:focus {
+  background-color: rgb(208, 208, 208);
+  border-style: none;
+}
+
+.btn-form {
+  text-align: center;
+}
+
+.btn-form button {
+  text-decoration: none;
+  border: 0.1rem solid rgb(62, 184, 184);
+  padding: 0.5rem 4rem;
+  border-radius: 2rem 0;
+  margin-inline: auto;
+}
+
+.btn-filled-form {
+  background-color: rgb(62, 184, 184);
+  color: white;
+}
+
+.btn-form button:hover {
+  box-shadow: -0.5rem 0.2rem 0.4rem rgba(133, 187, 187, 0.39);
+  background-color: rgba(0, 0, 0, 0.134);
+  cursor: pointer;
+}
+
+#map img {
+  margin-inline: auto;
+  height: 10rem;
+}
+
+footer {
+  background-color: rgba(88, 88, 88, 0.433);
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  gap: 2rem;
+  padding-block: 0.3rem;
+}
+
+.icon {
+  height: 2rem;
+}
+
+.pictures {
+  display: grid;
+  grid-template-columns: repeat(8, 1fr);
+  grid-template-rows: repeat(3, 10vw);
+  grid-gap: 0.3rem;
+}
+
+.pictures img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  margin-block-start: 1rem;
+}
+
+.pic1 {
+  grid-area: 1/1/2/4;
+}
+
+.pic2 {
+  grid-area: 2/1/3/2;
+}
+
+.pic3 {
+  grid-area: 2/2/3/4;
+}
+
+.pic4 {
+  grid-area: 3/1/4/3;
+}
+
+.pic5 {
+  grid-area: 3/3/4/4;
+}
+
+.pic6 {
+  grid-area: 1/4/3/5;
+}
+
+.pic7 {
+  grid-area: 3/4/4/5;
+}
+
+.pic8 {
+  grid-area: 1/5/4/7;
+}
+
+.pic9 {
+  grid-area: 1/7/2/9;
+}
+
+.pic10 {
+  grid-area: 2/7/3/8;
+}
+
+.pic11 {
+  grid-area: 3/7/4/8;
+}
+
+.pic12 {
+  grid-area: 2/8/4/9;
```

Comment:
Există o proprietate [`grid-auto-flow`](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-auto-flow) care s-ar putea să te ajute. În loc să calculezi rândul și coloana, pui browserul la treabă și pe fiecare imagine definești doar câte rânduri și coloane se întinde, cu `span`.

Cu asta, am ajuns foarte aproape de rezultatul tău:

```css
.pictures {
  grid-auto-flow: column dense;
}

.pic12 {
  /* Am setat pe fiecare imagine exact dimensiunile folosite de tine */
  grid-area: span 2 / span 1;
}
```

Reply:
Aceeași chestie și pe paginile cu destinații.

Reply:
La destinații nu știu de ce nu am reușit. Fiecare destinație arată diferit imaginile (nu rămân pe aceeași poziție). În codul modificat am păstrat prima variantă.